docs/_data/australia-2020-config.yaml

```diff
@@ -17,12 +17,12 @@ buttons:
   top:
      # - text: The conference is moving online
      #   link_absolute: /blog/moving-portland-prague-2020-online/
-      # - text: Buy a Ticket!
-     #   link: /tickets
+    - text: Buy a Ticket!
+      link: /tickets
     - text: Sponsor us
       link: /sponsors/prospectus
-    #  - text: Submit a Talk
-    #    link: /cfp
+    - text: Submit a Talk
```

Comment:
Having three top buttons kinda breaks the layout - we only support two.
![Screenshot 2020-07-21 at 11 19 24](https://user-images.githubusercontent.com/98594/88036600-10e6d780-cb44-11ea-9d22-6110b37be693.png)

Reply:
@mxsasha - I can check out Ti.to (I have access) if you want. 

I'd like to use the grant program if that's ok - I realised we need to set up an Australian form for this. Can you set one up?

Reply:
@mxsasha - I'll turn off the Sponsor us button from the top menu.

Reply:
I can't figure out how to give you access to the tito event - @thatdocslady any idea?
I've created the grant program form and updated the dates to something reasonable in 9a85f86.

Also, somehow flaglanding is still True in my latest checkout, and on the readthedocs build? But I saw a commit of you disabling it, must have snuck in again.